packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -2030,19 +2024,13 @@ namespace Tpetra {
     this->setLocallyModified ();
 
 #ifdef HAVE_TPETRA_DEBUG
```

Comment:
This looks interesting.  What happened here?

Reply:
@mhoemmen Some variables that were only used inside HAVE_TPETRA_DEBUG blocks were declared outside of them.